openedx/features/pakx/lms/overrides/utils.py

```diff
@@ -313,3 +312,30 @@ def get_date_diff_in_days(future_date):
 
     current_date = datetime.now().date()
     return (future_date.date() - current_date).days
+
+
+def validate_text(text):
```

Comment:
I think we should revise this method's name, `validate_text` doesn't tell it is validating emoticons or special characters.